src/test/java/com/example/PascalTriangleTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import static com.example.PascalTriangle.*;
```

Comment:
Do we need this `import` at all? Also it is a discouraged to use wildcard imports.

Reply:
I have a thing for imports... The code turns to be simplier without drawing PascalTriangle class before all 5 method calls. However, I totally agree with issue of wildcard - wanted to show off before my teacher :) Willing to push a new version soon.